src/x86_const.rs

```diff
@@ -270,3 +270,12 @@ pub enum InsnSysX86 {
     SYSCALL = InsnX86::SYSCALL as isize,
     SYSENTER = InsnX86::SYSENTER as isize,
 }
+
+#[repr(C)]
+#[derive(PartialEq, Debug, Clone, Copy)]
+pub struct X86Mmr {
+    pub selecter: u64,
```

Comment:
typo: selector

Reply:
typo fixed in https://github.com/ekse/unicorn-rs/pull/55/commits/d9795935e4c02c8e8f31ea05953818ab989e4b47